src/audio_listener.rs

```diff
@@ -112,3 +122,54 @@ fn constructor(ctx: CallContext) -> Result<JsUndefined> {
 
     ctx.env.get_undefined()
 }
+
+#[js_function(3)]
+fn set_position(ctx: CallContext) -> Result<JsUndefined> {
+    // TODO https://webaudio.github.io/web-audio-api/#dom-audiolistener-setposition
+    //
+    // When any of the positionX, positionY, and positionZ AudioParams for this AudioListener have
+    // an automation curve set using setValueCurveAtTime() at the time this method is called, a
+    // NotSupportedError MUST be thrown.
```

Comment:
I think this is actually ok as the error will be thrown by the `set_value` call. The check is done in the `AudioParam`.

But this is also something that should be fixed on the rust side at some point because the `panic` would occur in the render thread which is not good...